components/list/bullet/src/index.js

```diff
@@ -5,9 +5,10 @@ import cx from 'classnames'
 
 const BASE_CLASS = 'sui-ListBullet'
 
-const ListBullet = ({listItems, responsive}) => {
+const ListBullet = ({listItems, responsive, smallFont}) => {
   const listBulletClass = cx(BASE_CLASS, {
-    [`${BASE_CLASS}--responsive`]: responsive
+    [`${BASE_CLASS}--responsive`]: responsive,
```

Comment:
Are two variants compatible?

Reply:
We does not have specification about it  :/